MapboxCoreNavigation/FreeDriveLocationManager.swift

```diff
@@ -0,0 +1,134 @@
+import MapboxNavigationNative
+import CoreLocation
+import MapboxDirections
+import MapboxAccounts
+
+public protocol FreeDriveDebugInfoListener: AnyObject {
```

Comment:
Rename this protocol to FreeDrivingDebuggingInfoDelegate or somesuch. Listeners are called observers or delegates in Swift.